spec/18-namespaces.md

```diff
@@ -135,24 +137,32 @@ namespace NS3\Sub1;
   <i>namespace-function-or-const:</i>
     function
     const
+
+  <i>namespace-use-group-clauses:</i>
+    <i>namespace-use-group-clause</i>
+    <i>namespace-use-group-clauses</i>  ,  <i>namespace-use-group-clause</i>
+
+  <i>namespace-use-group-clause:</i>
+    <i>name</i>  <i>namespace-aliasing-clause<sub>opt</sub></i>
```

Comment:
This can have a `namespace-function-or-const` prefix. However only if there is no prefix on the use itself. So this should be split up into two separate definitions for group use (prefix before use) and mixed group use (prefix before use elements).